app/views/screens/AccountScreen.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { StyleSheet, Text } from "react-native";
+
+import Screen from "../components/Screen";
+import defaultStyle from "../styles/styles";
+import colors from "../styles/colors";
+
+function AccountScreen(props) {
+  return (
+    <Screen style={styles.container}>
+      <Text style={styles.text}>Account screen coming soon...</Text>
+    </Screen>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    paddingHorizontal: 20,
+    width: "100%",
+    backgroundColor: colors.sb_blue_100,
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  text: {
+    ...defaultStyle.text,
```

Comment:
If we add this at App level then I do not think we need this line in every component, it will inherit from root/parent component. ---> Moving towards standardization

Reply:
Inheritance does not work in react-native app
<img width="855" alt="image" src="https://github.com/ndedhia1210/KiranaShop/assets/14354751/13605839-7b24-4a9d-81b7-554f117ace8f">

Reply:
Actually this ^^ is the reason why we have the defaultStyles concept.